### PR TITLE
Adding support for parsing ERROR blocks in MAP-Errors

### PR DESCRIPTION
**Pull request by shokuie**
As Errors required along Operations for any MAP implementation and for the sake of better coverage, I've added what it requires to parse MAP-Errors.EXP

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -266,20 +295,25 @@ function parse(s) {
...
-  const opRe = /\b([\w-]+) OPERATION::=/g;
+  const opRe = /\b([\w-]+) (OPERATION|ERROR)::=/g;
   let match = null;
 
-  const operations = {};
+  const blocks = {};
 
   while (match = opRe.exec(s)) {
-    let operationName = match[1];
+    let operationErrorName = match[1];
```
Comment (atesgoral) [inline index.js:304]: Maybe call this `blockName`?

**Review by atesgoral — APPROVED** [review]
LGTM besides that minor comment about the variable name!

File: index.js (hunk 1 above)
Comment (atesgoral) [reply index.js:304]: I'll do it while cutting a new version!

**Comment by atesgoral** [comment]
https://www.npmjs.com/package/map-modules v0.1.0 now has MAP-Errors, and also the source ASN.1 files have been updated from version e30 to f30: https://github.com/atesgoral/map-modules/blob/master/dist/MAP-Errors.EXP.json

**Comment by atesgoral** [comment]
Speaking of errors, the ERRORS blocks in operations themselves, they're not as important, or is it worth tackling them at some point as well?

**Comment by shokuie** [comment]
👍 @atesgoral. Regarding the ERRORS block, it would be another step forward. Might take a look into it but it's not as required as parsing the MAP-Error module.

**Pull request merged**